cargo_config.in

```diff
@@ -0,0 +1,3 @@
+[target.@TARGET@]
```

Comment:
@bharrisau This is why the `platform` -> `target` mapping is currently done in configure. If we move it to the Makefile, then we have to manually define all the targets in this file... and then I don't know how I would go about looking for every toolchain.